nifi-commons/nifi-processor-utilities/pom.xml

```diff
@@ -45,5 +45,16 @@
             <artifactId>nifi-ssl-context-service-api</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>io.thekraken</groupId>
+            <artifactId>grok</artifactId>
+            <version>0.1.4</version>
```

Comment:
A new version has been released today and contains important fixes (reduced depencies, better feature parity with logstash, etc). May I  suggest we upgrade the  dependency?